makefile.gen

```diff
@@ -159,12 +160,15 @@ cleanAsm: cleanasm
 .PHONY: cleanRelease cleanDebug cleanAsm
 
 
-$(OUT)/rom.bin: $(OUT)/rom.out
+injectSymbolsInROM:	$(OUT)/rom.bin $(OUT)/rom.out $(OUT)/symbol.txt
+	$(CONVSYM) $(OUT)/symbol.txt $(OUT)/rom.bin -in txt -inopt "/fmt='%X %*[TtBbCcDd] %511s' /offsetFirst+" -range 0 FFFFFF -a -ref @MDDBG__SymbolDataPtr

```

Comment:
It looks like the "/fmt='%X %*[TtBbCcDd] %511s' /offsetFirst+" parameter is not properly translated when executed from make. It generated an error in the debug build process :
```
WARNING: Illegal characters found while parsing option string
ERROR: Unknown option: /Apps/SGDK/fmt, skipping
FATAL: Couldn't resolve symbol "MDDBG__SymbolDataPtr"
```

Look like the / is automatically prefixed by SGDK path, not sure why this is happening but still need to be sorted out, i'm currently investigating..

Reply:
Which command line interpreter do you use? I had no issues running this with make on my system.

Reply:
Good catch for the fix, it works perfectly now :-)